whylog/config/__init__.py

```diff
@@ -11,7 +16,9 @@ def create_investigation_plan(self, front_input):
 
 class YamlConfig(AbstractConfig):
```

Comment:
In the future we may want a `JsonConfig` or `MySQLConfig`
Please consider which methods should go up to `AbstractConfig` or `AbstractFileConfig` - YamlConfig should only contain methods specific to the _yaml_ format specifically.